templates/python/build.sh

```diff
@@ -1 +1,8 @@
-#!/bin/bash
\ No newline at end of file
+#!/bin/bash
+
+python_templates=$(ls -d */ | sed 's/\//\n/g')
+current_directory=`pwd`
+for template in $maven_templates; do
+   cd $current_directory/$template
+   echo "Building $template template"
+done
```

Comment:
Why do we need this at all?

Reply:
It is not doing anything other than looping over the folders we can add the commands to build the python templates in there so we check everything can be built